src/crud.js

```diff
@@ -33,32 +33,35 @@ models: {
 
 export default (server, model, { prefix, defaultConfig: config, models: permissions }) => {
   const modelName = model._singular;
+  const permissionsIsArray = Array.isArray(permissions);
```

Comment:
We're not throwing an error in case of permissions not being an array, are we?
What do you think about this?

```javascript
if (typeof permissions !== 'undefined' && !Array.isArray(permissions) {
  throw new Error("models property must be an array");
}
```